crates/astria-celestia-jsonrpc-client/k8s/scripts/start-bridge.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -x
+
+set -o errexit -o nounset -o pipefail
+
+if genesis_hash=$(curl -s -S -X GET "http://127.0.0.1:26657/block?height=1" | jq -er '.result.block_id.hash');
```

Comment:
where is the `26657` port from? looks like this is hitting the default tendermint endpoint celestia is using?

Reply:
This is from here (archived sequencer-relayer repo): https://github.com/astriaorg/sequencer-relayer/blob/master/docker/start-bridge.sh

:-)